activestorage/app/models/active_storage/blob.rb

```diff
@@ -110,6 +110,8 @@ def text?
   # Hiding the +service_url+ behind a redirect also gives you the power to change services without updating all URLs. And
   # it allows permanent URLs that redirect to the +service_url+ to be cached in the view.
   def service_url(expires_in: service.url_expires_in, disposition: :inline, filename: self.filename, **options)
```

Comment:
Now that we fall back to `self.filename` in the method body, make `nil` the default in the formal argument list to eliminate the duplication:

```ruby
def service_url(expires_in: service.url_expires_in, disposition: :inline, filename: nil, **options)
```

Reply:
👍